prawns/governance/schema.graphql

```diff
@@ -53,9 +53,25 @@ type SubstrateDemocracyProposal @entity {
   date: DateTime!
   proposalIndex: Int! @index
   amount: BigInt!
+  status: String!
+  updatedAt: DateTime!
+  tabledAtBlock: BigInt @index
+  referendaId: String @index
```

Comment:
You should name this, for consistency, `democracyReferendaId`, I think :D

Reply:
Good point!